fluent-bit/coralogix/README.md

```diff
@@ -8,18 +8,49 @@ The default values we provide can be overriden according to your needs, the defa
 helm show values coralogix-charts-virtual/fluent-bit-coralogix
 ```
 
-## Installation
+## Installation with default/dynamic app_name and sub_system
+Dynamic App_Name and Sub_System means taking the value from the running containers, so it can be 'namespace', 'pod_name', 'container_name'.
```

Comment:
it can be any accessible environment variable such as:
1. https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
2. Any other supplied environment variable